content/fr/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -24,9 +24,9 @@ _EndpointSlices_ offrent une méthode simple pour suivre les Endpoints d'un rés
 ## Resource pour EndpointSlice {#endpointslice-resource}
 
 Dans Kubernetes, un EndpointSlice contient des reférences à un ensemble de Endpoints. 
```

Comment:
Actually:
```suggestion
Dans Kubernetes, un EndpointSlice contient des reférences à un ensemble de points finals (_endpoints_) du réseau cluster. 
```
?

(Please feel free to reword into idiomatic French, if needed).

Reply:
"points finals" should be rather replaced by "terminaisons" which is the best translation of "endpoints", I just did an additional commit for that.